### PR TITLE
Update makefile

### PR DESCRIPTION
**Pull request by dacianf**
It still fails says on line 60 recipe for target sound.o and amc.o failed

### CODE REVIEW

**Comment by waddlesplash** [comment]
So is this only a partial fix?

**Comment by dacianf** [comment]
yes, i'm trying to make a recipe file for this in haiku and this is a partial fix,

**Comment by waddlesplash** [comment]
Can you try to fix it the rest of the way?

**Comment by dacianf** [comment]
I don't really know what to do more...

**Comment by richienyhus** [comment]
@dacianf You will need to change amc/src/midi-keyb.h

File: amc/src/makefile [hunk 1]
```diff
@@ -20,47 +20,44 @@ OBJS = amc.o \
...
+    set_attr amc
+#   xres -o amc amc.rsrc
+#   mimeset -f amc
 
 install:
-	ln -sf $(PWD)/amc /boot/apps
-	ln -sf $(PWD)/amc /boot/home/config/be/Applications
-	
+    ln -sf $(PWD)/amc /boot/apps
+    ln -sf $(PWD)/amc /boot/home/config/be/Applications
```
Comment (richienyhus) [inline amc/src/makefile:54]: Hasn't this changed with package management?

**Pull request closed without merging**